monkey/infection_monkey/post_breach/actions/clear_command_history.py

```diff
@@ -0,0 +1,49 @@
+import subprocess
+
+from common.data.post_breach_consts import POST_BREACH_CLEAR_CMD_HISTORY
+from infection_monkey.post_breach.clear_command_history.clear_command_history import \
+    get_commands_to_clear_command_history
+from infection_monkey.post_breach.pba import PBA
+from infection_monkey.telemetry.post_breach_telem import PostBreachTelem
+
+
+class ClearCommandHistory(PBA):
+    def __init__(self):
+        super().__init__(name=POST_BREACH_CLEAR_CMD_HISTORY)
+
+    def run(self):
+        results = [pba.run() for pba in self.clear_command_history_PBA_list()]
+        PostBreachTelem(self, results).send()
+
+    def clear_command_history_PBA_list(self):
+        return self.CommandHistoryPBAGenerator().get_clear_command_history_pbas()
```

Comment:
You think we can cover this with UTs if you mock get_commands_to_clear_command_history (after you address my comments there as well)?

Reply:
Would probably be a little complicated since there are nested classes involved, but we can, yes.

Reply:
I'll give it a shot after updating the other 3 techniques' PRs.